tests/tests.yaml

```diff
@@ -28,7 +28,7 @@ answer_tests:
     - yt/frontends/athena/tests/test_outputs.py:test_blast
     - yt/frontends/athena/tests/test_outputs.py:test_stripping
 
-  local_athena_pp_004:
+  local_athena_pp_006:
     - yt/frontends/athena_pp/tests/test_outputs.py:test_disk
```

Comment:
I think we actually need to comment this line out, see
https://tests.yt-project.org/job/yt_py38_git/4630/testReport/junit/nose.failure/Failure/runTest/